docs/docker.md

```diff
@@ -21,7 +21,8 @@ Feel free to comment out the apps you do not need.
 Create a configuration directory on your host machine:
 ```
 git clone -b docker-master https://github.com/georchestra/datadir.git
-sudo mv datadir /etc/georchestra
+sudo mkdir /etc/georchestra
+sudo mv datadir/* /etc/georchestra/.
```

Comment:
i understand... a solution could consist in downloading files in the right directory, like that:
sudo cd /etc
sudo git clone -b docker-master https://github.com/georchestra/datadir.git georchestra
if the georchestra directory already exists, the download is stopped, with this warning:
fatal: le chemin de destination 'georchestra' existe déjà et n'est pas un répertoire vide.
hint:
you should have a git prompt if you want to know the branch on which you are.
cat /root/.bashrc
# http://stackoverflow.com/questions/15883416/adding-git-branch-on-the-bash-command-prompt
source /usr/local/bin/git-prompt.sh
PS1='[\u@\h \W$(__git_ps1 " (%s)")]\$ '